AdaptiveCards/templating/language.md

```diff
@@ -14,19 +14,27 @@ Templating enables the separation of **data** from **layout** in your Adaptive C
 
 > [!IMPORTANT] 
 > 
-> These features are **in preview and subject to change**. Your feedback is not only welcome, but  critical to ensure we deliver the features **you** need.
+> **Breaking changes** in the **May 2020 Release Candidate**
+>
+> We've been hard at work getting templating released, and we're finally in the home stretch! We had to make some minor breaking changes as we close on the release.
 
-When authoring a template you can specify the data inline with the `AdaptiveCard` payload, or at runtime using the [Templating SDKs](sdk.md).
+## Breaking changes as of May 2020
 
-## Specify data within the card
+1. The binding syntax changed from `{...}` to `${...}`
+    * For Example: `"text": "Hello {name}"` becomes `"text": "Hello ${name}"`
+    
+## Binding to data
 
-To provide data directly within the card payload, simply add a `$data` attribute to your `AdaptiveCard` (seen below).
+When authoring a template you have two options to provide to relevant data.
 
-## Binding to the data
+1. **Option A: Inline with the template payload**. You can provide the data inline with the `AdaptiveCard` template payload. To do so, simply add a `$data` attribute to the root `AdaptiveCard` object.
+2. **Option B: As a separate data object**. with this option you provide two separate objects to the [Templating SDK](sdk.md) at runtime: the template and the data. This will be the more common approach, since typically you will create a template and want to provide dynamic data later.
+
+### Optioan A: Inline data
 
 You can bind to the data within the `body` or `actions` of the card.
 
```

Comment:
I feel like this paragraph makes a jump out of nowhere, probably adding a subtitle similar to "Features supported" or some kind of introduction may make more sense